include/admin-switcher.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+
+class PLL_Admin_Switcher extends PLL_Switcher {
+
+	/**
+	 * Get the language elements for use in a walker
+	 *
+	 * @since 1.2
+	 *
+	 * @param PLL_Frontend_Links $links Instance of PLL_Frontend_Links.
+	 * @param array              $args  Arguments passed to {@see PLL_Switcher::the_languages()}.
+	 * @return array Language switcher elements.
+	 */
+	protected function get_elements( $links, $args ) {
```

Comment:
Much of `PLL_fronted_Switcher::get_elements()` and `PLL_Admin_Switcher::get_elements()` is quite similar, we could share this method from the parent class. See proposed changes below.